framework/include/mfem/kernels/MFEMVectorFEDivergenceKernel.h

```diff
@@ -0,0 +1,32 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifdef MFEM_ENABLED
+
+#pragma once
+#include "MFEMMixedBilinearFormKernel.h"
+
+/*
+(k div u, v)
```

Comment:
```suggestion
 * (k div u, v)
```

Reply:
Done, and changed all other kernels at fault as well.